util/serial/src/lib.rs

```diff
@@ -99,6 +97,16 @@ mod json_u64 {
         }
     }
 
+    impl TryFrom<&json_u128::JsonU128> for JsonU64 {
+        type Error = std::num::TryFromIntError;
```

Comment:
Lets try spelling this 
```suggestion
        type Error = core::num::TryFromIntError;
```
I think that should fix the build

Reply:
this fixed the issue